clusterloader2/pkg/measurement/util/runtimeobjects/replicaswatcher.go

```diff
@@ -150,7 +154,14 @@ func (n *NodeCounter) shouldRun(obj interface{}) (bool, error) {
 		return false, fmt.Errorf("unexpected type of obj: %v. got %T, want *corev1.Node", obj, obj)
 	}
 	matched, err := podMatchesNodeAffinity(n.affinity, node)
-	return !node.Spec.Unschedulable && matched, err
+	tolerate := true
```

Comment:
Can we add an unit test that covers this change?

Reply:
ok, have any suggestions about testcase?

Reply:
Maybe something like: a pod that wants to run on a given node, without toleration it wouldn't be possible (test case 1), but it has toleration it's OK (test case 2)